api/src/main/java/gov/cms/ab2d/api/controller/common/ApiText.java

```diff
@@ -0,0 +1,57 @@
+            " for subsequent status requests (polling location)";
+    public static final String BULK_SINCE = "Beginning time of query. Returns all records \"since\" this time. At this time, it must be after " + SINCE_EARLIEST_DATE;
+    public static final String BULK_RESPONSE_LONG = "Absolute URL of an endpoint for subsequent status requests (polling location)";
+    public static final String EXPORT_STARTED = "Export request has started";
+    public static final String JOB_COMPLETE = "The job is complete.";
+    public static final String STATUS_DELAY = "A delay time in seconds before another status request will be accepted.";
+    public static final String PROGRESS = "Completion percentage, such as 50%";
+    public static final String FILE_EXPIRES = "Indicates when (an HTTP-date timestamp) the files " +
+            "listed will no longer be available for access.";
+    public static final String CANCEL = "Cancel Export Job";
+    public static final String AUTH = "Authorization";
+    public static final String CAP_DESC = "Returns the FHIR capability statement";
+    public static final String CAP_REQ = "A request for the FHIR capability statement";
+    public static final String JSON = "application/json";
+    public static final String DOWNLOAD = "Downloads Export File";
+    public static final String DOWNLOAD_DESC = "Downloads a file produced by an export job.";
+    public static final String CONTENT_TYPE = "Content-Type";
+    public static final String CONTENT_TYPE_DESC = "Content-Type header that matches the file format being delivered: ";
+    public static final String BULK_DNLD = "Bulk Data File Download API";
+    public static final String BULK_DNLD_DSC = "After creating a job, the API to download the generated bulk download files";
+    public static final String NOT_FOUND = "Job or file not found. ";
+    public static final String STILL_RUNNING = "The job is still in progress.";
+    public static final String STATUS_DES = "Returns a status of an export job.";
+    public static final String JOB_ID = "A job identifier";
+    public static final String FILE_NAME = "A file name";
+    public static final String DNLD_DESC = "Returns the requested file as " + NDJSON_FIRE_CONTENT_TYPE + " or " + ZIPFORMAT;
+    public static final String STATUS = "Status of export job";
+    public static final String JOB_NOT_FOUND = "Job not found. " + GENERIC_FHIR_ERR_MSG;
+    public static final String JOB_CANCELLED_MSG = "Job canceled";
+    public static final String CAP_STMT = "FHIR capability statement";
+    public static final String CAP_API = "Provides the standard required capability statement";
+    public static final String CAP_RET = "FHIR Capability Statement Returned";
+    public static final String CONTRACT_NO = "A contract number";
+    public static final String EXPORT = "Export Claim Data";
+    public static final String CONT_LOC = "Content-Location";
+    public static final String ASYNC = "respond-async";
```

Comment:
Might be a standard constant for this one in HttpHeaders.

If not, how about RESPOND_ASYNC?